components/Common/OramaSearchBar/OramaLogo.tsx

```diff
@@ -0,0 +1,731 @@
+.st8{clip-path:url(#SVGID_00000045603176312755833640000016830543301894725252_);fill-rule:evenodd;clip-rule:evenodd;fill:url(#SVGID_00000055669823316971348430000007456605679816995478_);}
+.st9{clip-path:url(#SVGID_00000050637087109987908300000001655297821271179658_);fill-rule:evenodd;clip-rule:evenodd;fill:url(#SVGID_00000121276425135176461840000001411911869491139213_);}
+.st10{clip-path:url(#SVGID_00000155862216285633165090000011392877503296698766_);fill-rule:evenodd;clip-rule:evenodd;fill:url(#SVGID_00000028307555131462038650000001206007083441086101_);}
+`;
+
+const logoLightStyle = `
+.st0{fill:#3A393D;}
+.st1{clip-path:url(#SVGID_00000121237917302734654570000007802433009826490514_);fill:#FB81B8;}
+.st2{clip-path:url(#SVGID_00000121237917302734654570000007802433009826490514_);fill-rule:evenodd;clip-rule:evenodd;fill:url(#SVGID_00000067951042823557912200000003375363423373398917_);}
+.st3{clip-path:url(#SVGID_00000121237917302734654570000007802433009826490514_);fill:url(#SVGID_00000173158817079393216730000015416942586975534773_);}
+.st4{clip-path:url(#SVGID_00000121237917302734654570000007802433009826490514_);fill-rule:evenodd;clip-rule:evenodd;fill:url(#SVGID_00000093858169607081702240000004176952752693074346_);}
+.st5{clip-path:url(#SVGID_00000121237917302734654570000007802433009826490514_);fill-rule:evenodd;clip-rule:evenodd;fill:url(#SVGID_00000070090918966372163940000003253904251979675325_);}
+.st6{clip-path:url(#SVGID_00000121237917302734654570000007802433009826490514_);fill-rule:evenodd;clip-rule:evenodd;fill:url(#SVGID_00000132776349315605697860000002688660824302203828_);}
+.st7{clip-path:url(#SVGID_00000121237917302734654570000007802433009826490514_);fill-rule:evenodd;clip-rule:evenodd;fill:#F97CBF;}
+.st8{clip-path:url(#SVGID_00000121237917302734654570000007802433009826490514_);fill-rule:evenodd;clip-rule:evenodd;fill:url(#SVGID_00000016755478859824984880000007136540152028842637_);}
+.st9{clip-path:url(#SVGID_00000121237917302734654570000007802433009826490514_);fill-rule:evenodd;clip-rule:evenodd;fill:url(#SVGID_00000016785315492321026870000005125100075421436076_);}
+.st10{clip-path:url(#SVGID_00000121237917302734654570000007802433009826490514_);fill-rule:evenodd;clip-rule:evenodd;fill:url(#SVGID_00000164475283253175513550000014604273281455785608_);}
+`;
+
+// Keep up to date with: https://docs.oramasearch.com/logo/logo-orama-dark.svg
+const OramaLogoDark = ({ className }: OramaLogoProps): JSX.Element => {
+  return (
+    <svg
+      version="1.1"
+      id="Layer_1"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      x="0px"
+      y="0px"
+      viewBox="0 0 4207.11 1155.11"
+      className={className}
+      style={{ enableBackground: 'new 0 0 4207.11 1155.11' } as any}
+      xmlSpace="preserve"
+    >
+      <style
+        type="text/css"
+        dangerouslySetInnerHTML={{ __html: logoDarkStyle }}
```

Comment:
Could we move the styles to a proper css file so we don't have to use `dangerouslySetInnerHTML`

Reply:
I tried. And I discover none of the styles were actually used. Lol.
I've removed everything.